apis/r/R/SOMAExperimentAxisQuery.R

```diff
@@ -705,6 +705,259 @@ SOMAExperimentAxisQuery <- R6::R6Class(
       SeuratObject::DefaultAssay(mat) <- private$.measurement_name
       validObject(mat)
       return(mat)
+    },
+    #' @description Loads the query as a
+    #' \code{\link[SingleCellExperiment]{SingleCellExperiment}} object
+    #'
+    #' @param X_layers \Sexpr[results=rd]{tiledbsoma:::rd_outgest_xlayers('sce')}
+    #' @param obs_index \Sexpr[results=rd]{tiledbsoma:::rd_outgest_index('sce')}
+    #' @param var_index \Sexpr[results=rd]{tiledbsoma:::rd_outgest_index('sce', 'var')}
+    #' @param obsm_layers \Sexpr[results=rd]{tiledbsoma:::rd_outgest_mlayers('sce')}
+    #' @param obs_column_names \Sexpr[results=rd]{tiledbsoma:::rd_outgest_mdnames('sce')}
+    #' @param var_column_names \Sexpr[results=rd]{tiledbsoma:::rd_outgest_mdnames('sce', 'var')}
+    #' @param obsp_layers \Sexpr[results=rd]{tiledbsoma:::rd_outgest_players('sce')}
+    #' @param varp_layers \Sexpr[results=rd]{tiledbsoma:::rd_outgest_players('sce', 'varp')}
+    #'
+    #' @return A \code{\link[SingleCellExperiment]{SingleCellExperiment}} object
+    #'
+    to_single_cell_experiment = function(
+      X_layers = NULL,
+      obs_index = NULL,
+      var_index = NULL,
+      obs_column_names = NULL,
+      var_column_names = NULL,
+      obsm_layers = NULL,
+      obsp_layers = NULL,
+      varp_layers = NULL
+    ) {
+      .check_sce_installed()
+      stopifnot(
+        "'X_layers' must be a character vector" = is_character_or_null(X_layers),
+        "'obs_index' must be a single character value" = is.null(obs_index) ||
+          (is_scalar_character(obs_index) && !is.na(obs_index)),
+        "'var_index' must be a single character value" = is.null(var_index) ||
+          (is_scalar_character(var_index) && !is.na(var_index)),
+        "'obs_column_names' must be a character vector" = is.null(obs_column_names) ||
+          is.character(obs_column_names) ||
+          is_scalar_logical(obs_column_names),
+        "'var_column_names' must be a character vector" = is.null(var_column_names) ||
+          is.character(var_column_names) ||
+          is_scalar_logical(var_column_names),
+        "'obsm_layers' must be a character vector" = is.null(obsm_layers) ||
+          is.character(obsm_layers) ||
+          is_scalar_logical(obsm_layers),
+        "'obsp_layers' must be a character vector" = is.null(obsp_layers) ||
+          is.character(obsp_layers) ||
+          is_scalar_logical(obsp_layers),
+        "'varp_layers' must be a character vector" = is.null(varp_layers) ||
+          is.character(varp_layers) ||
+          is_scalar_logical(varp_layers)
+      )
+      pad_names <- function(x) {
+        stopifnot(
+          is.character(x)
+        )
+        if (is.null(names(x))) {
+          return(stats::setNames(nm = x))
+        }
+        unnamed <- !nzchar(names(x))
+        names(x)[unnamed] <- x[unnamed]
+        return(x)
+      }
+      # Because SingleCellExperiment is broken
```

Comment:
I know you know what this means

Please add a sentence or two for the benefit of the rest of us :)